test/integration/v2/v2_docker_registry_test.go

```diff
@@ -240,8 +246,15 @@ func TestV2RegistryGetTags(t *testing.T) {
 	if len(otherStream.Status.Tags) != 1 {
 		t.Errorf("expected 1 tag, got %#v", otherStream.Status.Tags)
 	}
-	history, ok := otherStream.Status.Tags[imageapi.DefaultImageTag]
-	if !ok {
+
+	var history *imageapiv1.NamedTagEventList
+	for i := range otherStream.Status.Tags {
```

Comment:
mildly interesting.  the Tags data structures were not the same between the internal and external apis so i had to this instead.